docs/minishift-installation/master.adoc

```diff
@@ -8,6 +8,10 @@ include::topics/templates/document-attributes.adoc[]
 
 This guide walks you through the process of installing the {launcher} application to run on a local cloud as provisioned by a {OpenShiftLocal}. This includes link:https://www.openshift.org/minishift/[{Minishift}], an all-in-one VM that includes a community version of OpenShift Origin, or link:https://developers.redhat.com/products/cdk/overview/[{CDK}], a VM that includes OpenShift Container Platform.
 
+.The {launcher} Application
+The {launcher} application runs on OpenShift and provides a hassle-free way of creating functional example applications called missions. When using the {launcher} application, you choose the mission you want to generate, the specific runtime you want to use, and how you want to build and run the mission's code. The {launcher} application then generates your custom project, called a booster, and either launches it directly to the same OpenShift instance, or provides a downloadable ZIP version of the project.
```

Comment:
> run the mission's code

Terribly sorry I didn't catch this the first time around. We do not use Saxon genitives, and _run_ is considered informal in the context of executing code, so please change this formulation to:

_execute the code of the mission_

Thank you.

Reply:
Saxon genitive, wow :-) I'd like to point out that the sentence in question is already pretty long, so I'd be careful in making it even longer.

Reply:
In case we want to make the message clearer, I suggest breaking the sentence into bullet points.